helm/repositories.bzl

```diff
@@ -124,12 +125,9 @@ exports_files(["helm{ext}"])
         ext = ext,
     ))
 
-    platform = "{}_{}".format(repository_ctx.os.name, repository_ctx.os.arch)
-    platform_repo_name = _helm_bin_repo_name(platform)
```

Comment:
One of the bugs here (and I must have not been testing under bzlmod) is that we need to use the actual name from the repository context to properly capture the bzlmod repo path in the symlink.